Units/simple-nsis.d/expected.tags

```diff
@@ -0,0 +1,20 @@
+Answer	input.nsi	/^Var Answer$/;"	v
+UserName	input.nsi	/^Var UserName$/;"	v
+StartmenuFolder	input.nsi	/^Var StartmenuFolder$/;"	v
+UNINSTDIR	input.nsi	/^Var UNINSTDIR$/;"	v
+\x21Program Files	input.nsi	/^Section "!Program Files" SEC01$/;"	s
```

Comment:
! at the head of tag means displaying the section name with bold style.
I wonder this should be part of the section name or not.
https://nsis.sourceforge.io/Reference/Section